### ISSUE TITLE
cypress crashes trying to parse commit

### ISSUE THREAD

**Issue by arthens** [cypress-io/cypress#847]
This might be related/duplicate of https://github.com/cypress-io/cypress/issues/841, but:
- we are getting a different stack trace 
- we don't use GPG signatures
- it's happening on all users, no email change involved

### Environment

- Operating System: ubuntu
- Cypress Version: 1.0.2

### Is this a Feature or Bug?

Bug

### Current behavior:

cypress cli crashes in CI. 

### How to reproduce:

Haven't been able to reproduce it locally.

### Additional Info (images, stack traces, etc)

```
✔  Verified Cypress! /usr/local/lib/node_modules/cypress/dist/Cypress

Opening Cypress...
--
  | TypeError: Cannot read property '0' of null
  | at Function.module.exports.Commit.actor (/usr/local/lib/node_modules/cypress/dist/Cypress/resources/app/packages/server/node_modules/gift/lib/commit.js:176:56)
  | at Function.module.exports.Commit.parse_commits (/usr/local/lib/node_modules/cypress/dist/Cypress/resources/app/packages/server/node_modules/gift/lib/commit.js:139:21)
  | at /usr/local/lib/node_modules/cypress/dist/Cypress/resources/app/packages/server/node_modules/gift/lib/commit.js:99:39
  | at ChildProcess.exithandler (child_process.js:209:7)
  | at emitTwo (events.js:106:13)
  | at ChildProcess.emit (events.js:191:7)
  | at maybeClose (internal/child_process.js:877:16)
  | at Socket.<anonymous> (internal/child_process.js:334:11)
  | at emitOne (events.js:96:13)
  | at Socket.emit (events.js:188:7)
  | at Pipe._handle.close [as _onclose] (net.js:493:12)
```

**Comment by pawelgalazka** [comment]
Having same issue 😞

**Comment by pawelgalazka** [comment]
I'm wondering, is Cypress version control in npm actually works ? or when it's downloading Cypress it still takes the newest version somehow ? The issue started happening suddenly today and not sure if it is related to email or signature change.

**Comment by brian-mann** [comment]
This is a duplicate of #841.

Yes, version controlling with NPM definitely works. There are a gazillion tests, we use different versions on several internal projects, and Cypress will always print the version it installed via stdout. You can even force the CLI to download a different version and it will output a warning to you anytime it runs.

**Comment by pawelgalazka** [comment]
@brian-mann Is it possible that it could be temporary fixed to some version downgrade ? (can't see if `gift` was used before, no releases info in repo)

**Comment by fraserxu** [comment]
From #841 it was described as related to PGP signature parsing, but from this issue it clarify that it's not because of it, maybe worth put the context into that issue as it helps identify the issue?

**Comment by brian-mann** [comment]
@pawelgalazka changelog is here: https://docs.cypress.io/guides/references/changelog.html

Don't think downgrading will do anything, we've been using the same `gift` module for probably at least a year now.

**Comment by brian-mann** [comment]
@fraserxu its true the issue only described PGP signature parsing but we ended up ripping out `gift` in favor of a new module here: https://github.com/cypress-io/commit-info

**Comment by fraserxu** [comment]
Thanks @brian-mann ❤️ the quick fix.

**Comment by brian-mann** [comment]
Fixed in [`1.0.3`](https://docs.cypress.io/guides/references/changelog.html#1-0-3).
